src/sfctl/tests/request_generation_test.py

```diff
@@ -788,3 +788,14 @@ def paths_generation_helper(self):  # pylint: disable=too-many-statements
             'GET',
             '/Tools/Chaos',
             ['api-version=6.2'])
+
+        # Cluster Upgrade Commands:
+        self.validate_command(  # config-upgrade
```

Comment:
Please also include the other variables of the command in this test. Thanks!

Reply:
Nope, this is the only validation for this particular command. That this was not there before was a gap in testing

Reply:
but I don't see a point in adding other variables with default values..

Reply:
Done

Reply:
Thank you! :)